fe/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1545,5 +1625,188 @@ private void materializeInlineViewResultExprForCrossJoinOrCountStar(InlineViewRe
         }
     }
 
+    /**
+     ------------------------------------------------------------------------------
+     */
+    /**
+     * Push down predicates rules
+     */
+
+    private void pushDownPredicates(Analyzer analyzer, SelectStmt stmt) throws AnalysisException {
```

Comment:
Could add some comment to these functions?
There is no comment at all?

Reply:
ok, i will explain it.